_sass/everything.scss

```diff
@@ -276,17 +394,34 @@ footer {
     }
 
    .headline {
+     margin-bottom: 0;
      .pure-button {
        margin-top: 0;
+       margin-bottom: 50px;
      }
    }
+
+   .nav-menu {
+     padding: 0;
+   }
+
+   .logo {
+     padding: 10px 10px;
+   }
 }
 
 /*
  * -- TABLET (AND UP) MEDIA QUERIES --
  * -- Purecss md (768px) --
  */
-@media screen and (max-width: 48em) {
+@media screen and (min-width: 48em) {
+
+  .pure-u-xl-1-2:first-child > .l-box {
+    padding-left: 0;
```

Comment:
Oh yes! thanks for finding that. The problem here is that the grid's responsive breakpoints don't match up with the css's responsive breakpoints. I can't exactly remember why I decided to mess with the breakpoints, but definitely those two need to be synced up at some point.